api/client/client.go

```diff
@@ -76,12 +78,15 @@ func NewDriverClient(driverName, version string) (*Client, error) {
 // standard unix domain socket
 func GetSupportedDriverVersions(driverName, serverPath string) ([]string, error) {
 	// Get a client handler
-	sockPath := "unix://" + config.DriverAPIBase + driverName + ".sock"
-	volumeClient, err := NewClient(sockPath, "")
+	if serverPath == "" {
```

Comment:
since "unix://" and "osd.sock" is repeated more than once, make them constants and create a function for getting server path ?